lib/features/sign_up/presentation/sign_up_screen.dart

```diff
@@ -83,14 +82,8 @@ class _SignUpScreenState extends State<SignUpScreen> {
       body: BlocConsumer<SignUpBloc, SignUpState>(
         listener: (context, state) {
           if (state is SignUpLoadedState) {
-            final user = FirebaseAuth.instance.currentUser!;
-            Navigator.of(context).pushReplacement(
-              MaterialPageRoute(
-                builder: (_) => MainScreen(
-                  user: user,
-                  name: user.displayName,
-                ),
-              ),
+            context.router.replaceNamed(
+              Routes.bottomNav
```

Comment:
rename bottomNav to main

Reply:
renamed to main and change route to /main